scripts/download_data.py

```diff
@@ -116,6 +116,45 @@ def getMaxPos(self, chrom):
         return minMax.maxPos
 
 
+def fetchSequence(ac, startIndex=None, endIndex=None, as_list=False):
```

Comment:
Actually, why do we have the ``as_list`` argument here at all? We're only using this function in one place, and it uses the non-default value for this. It would seem simpler to just return a list in all situations, and use ``"".join(fetchSequence('NP_056374.2',0,10))`` for the doctests.